scripts/portal-upgrade.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e # exit on first error
+
+# put the server down for maintenance
+. ${cwd}/portal-down.sh
+
+# rebuild and restart all docker containers
+docker-compose build --no-cache
+docker-compose down
+docker-compose up -d
+
+# enable the server again
+. ${cwd}/portal-up.sh
```

Comment:
The same about all of these. Can we use a path, relative to the position of the script rather than the CWD? And then pass it to `docker-compose` or just `cd` to it and then `cd -`?

Reply:
responded in https://github.com/NebulousLabs/skynet-webportal/pull/371/files#r488060349